arches/app/templates/views/graph/graph-designer/graph-tree.htm

```diff
@@ -53,6 +53,9 @@
             <span style="font-weight:bold;" data-bind="text: ': ' + node.nodeid"></span>
             <!-- /ko -->
             <span class="spacer"></span>
+            <!-- ko if: graphTree.showIds-->
+            <i class="jstree-node-action-icon ion-ios-copy-outline" role="presentation" data-bind="click: graphTree.copyNodeId.bind(graphTree)" data-toggle="tooltip" data-placement="auto" data-container="body" data-original-title="{% trans "Copy UUID to Clipboard" %}"></i>
+            <!-- /ko -->
```

Comment:
The reason the tooltip isn't working is because the button is not actually rendered if showids is false - when the tooltips are all initialized.  Instead of encasing this in a ko block, it might be more effective to use the the visible binding - then the dom element will be available to the tooltip generator when it needs to be - just not visible to the user.  
```suggestion
           <i class="jstree-node-action-icon ion-ios-copy-outline" role="presentation" data-bind="click: graphTree.copyNodeId.bind(graphTree), visible: graphTree.showIds()" data-toggle="tooltip" data-placement="auto" data-container="body" data-original-title="{% trans "Copy UUID to Clipboard" %}"></i>
```

Reply:
Thanks! This has been committed